lib/requester/requester.js

```diff
@@ -99,17 +101,34 @@ Requester.prototype.request = function (item, cb, scope) {
                 },
                 responseCookies: cookies
             },
+            contentEncoding = response.responseHeaders && response.responseHeaders['Content-Encoding'],
+            encodedData = contentEncoding && (rawResponse ? rawResponse : util.getEncodedBody(
+                response.responseBody, contentEncoding)),
             responseObj = { // @todo get rid of jsonifyResponse
                 code: response.responseCode.code,
                 header: response.responseHeaders,
                 body: responseString,
                 stream: resBody,
                 cookie: undefined, // @todo get from cookieJar and implement loading/unloading in SDK
-                responseTime: responseTime
+                responseTime: responseTime,
+                encodedData: encodedData
             };
 
         cb.call(scope || this, null, response, actualRequest, responseObj, request);
-    });
+    })
+    .on('response', function(response) {
+        if(response.headers['content-encoding']) {
+            var chunks = [];
+            response.on('data', function(chunk) {
+                chunks.push(chunk);
+            })
+            .on('end', function() {
+                chunks.forEach(function(chunk) {
+                    rawResponse += util.arrayBufferToString(chunk);
```

Comment:
I am returning the rawResponse instead of calculating here, so that it could be displayed on the app later, calculation could be done on the `response` in `postman-collection`. Also almost all the api-related tools, display the encoded content for gzipped content.